go.mod

```diff
@@ -8,10 +8,10 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-bindata/go-bindata v3.1.2+incompatible
 	github.com/gonum/graph v0.0.0-20190426092945-678096d81a4b
-	github.com/openshift/api v0.0.0-20241001152557-e415140e5d5f
+	github.com/openshift/api v0.0.0-20241101202457-04eb3fd119d2
```

Comment:
ok, this also points to 4.18 branch (https://github.com/openshift/api/commits/release-4.18?after=a09992e80c6891cc0bfa2fa7ec6af42af37c452e+34)